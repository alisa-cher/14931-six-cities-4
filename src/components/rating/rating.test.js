import React from 'react';
import renderer from 'react-test-renderer';
import Rating from "./rating.jsx";

it(`Rating renders correctly`, () => {
  const tree = renderer
    .create(
        <Rating
          classNamePrefix={`some-prefix`}
          rating={2}
          isInteger={true}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
