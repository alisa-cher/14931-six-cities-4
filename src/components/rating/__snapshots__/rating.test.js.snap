// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating renders correctly 1`] = `
<div
  className="some-prefix__rating some-prefix"
>
  <div
    className="some-prefix__stars rating__stars"
  >
    <span
      style={
        Object {
          "width": "40%",
        }
      }
    />
    <span
      className="visually-hidden"
    >
      Rating
    </span>
  </div>
</div>
`;
